cpp/tests/prims/mg_count_if_e.cu

```diff
@@ -0,0 +1,373 @@
+    return data;
+  }
+
+  template <typename graph_view_type>
+  static auto column_property(raft::handle_t const& handle,
+                              graph_view_type const& graph_view,
+                              property_buffer_type& property)
+  {
+    auto output_property = cugraph::col_properties_t<graph_view_type, type>(handle, graph_view);
+    copy_to_adj_matrix_col(
+      handle, graph_view, cugraph::get_dataframe_buffer_begin(property), output_property);
+    return output_property;
+  }
+
+  template <typename graph_view_type>
+  static auto row_property(raft::handle_t const& handle,
+                           graph_view_type const& graph_view,
+                           property_buffer_type& property)
+  {
+    auto output_property = cugraph::row_properties_t<graph_view_type, type>(handle, graph_view);
+    copy_to_adj_matrix_row(
+      handle, graph_view, cugraph::get_dataframe_buffer_begin(property), output_property);
+    return output_property;
+  }
+};
+
+template <typename T>
+struct generate : public generate_impl<T> {
+  static T initial_value(int init) { return static_cast<T>(init); }
+};
+template <typename... Args>
+struct generate<std::tuple<Args...>> : public generate_impl<Args...> {
+};
+
+struct Prims_Usecase {
+  bool check_correctness{true};
+  bool test_weighted{false};
+};
+
+template <typename input_usecase_t>
+class Tests_MG_TransformCountIfE
+  : public ::testing::TestWithParam<std::tuple<Prims_Usecase, input_usecase_t>> {
+ public:
+  Tests_MG_TransformCountIfE() {}
+  static void SetupTestCase() {}
+  static void TearDownTestCase() {}
+
+  virtual void SetUp() {}
+  virtual void TearDown() {}
+
+  // Verify the results of count_if_e primitive
+  template <typename vertex_t,
+            typename edge_t,
+            typename weight_t,
+            typename result_t,
+            bool store_transposed>
+  void run_current_test(Prims_Usecase const& prims_usecase, input_usecase_t const& input_usecase)
+  {
+    // 1. initialize handle
+
+    raft::handle_t handle{};
+    HighResClock hr_clock{};
+
+    raft::comms::initialize_mpi_comms(&handle, MPI_COMM_WORLD);
+    auto& comm           = handle.get_comms();
+    auto const comm_size = comm.get_size();
+    auto const comm_rank = comm.get_rank();
+
+    auto row_comm_size = static_cast<int>(sqrt(static_cast<double>(comm_size)));
+    while (comm_size % row_comm_size != 0) {
+      --row_comm_size;
+    }
+    cugraph::partition_2d::subcomm_factory_t<cugraph::partition_2d::key_naming_t, vertex_t>
+      subcomm_factory(handle, row_comm_size);
+
+    // 2. create MG graph
+
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      hr_clock.start();
+    }
+    auto [mg_graph, d_mg_renumber_map_labels] =
+      input_usecase.template construct_graph<vertex_t, edge_t, weight_t, store_transposed, true>(
```

Comment:
I just changed this in a PR that got merged. If you merge the latest 21.10 you'll find this doesn't compile.

This call should be something like:
`cugraph::test::construct_graph<vertex_t, edge_t, weight_t, store_transposed, true>(handle, input_usecase, prims_usecase.test_weighted, true);`